frame/support/src/weights.rs

```diff
@@ -320,6 +320,8 @@ pub struct DispatchInfo {
 	pub class: DispatchClass,
 	/// Does this transaction pay fees.
 	pub pays_fee: Pays,
+	/// PoV size used by this transaction.
+	pub pov_size: u32,
```

Comment:
related to Basti question https://github.com/paritytech/substrate/pull/9763#issuecomment-918458515
Here is should be written what is the PoV size, either it correspond to the compressed size of the PoV for the given dispatchable or the non-compressed one.